### DISCUSSION TITLE
Failed to connect SCTP socket error= network is unreachable

### DISCUSSION THREAD

**Discussion by rajindersin** [srsran/srsRAN_Project#322] [General Help] [answered]
Hi 
I configured open5GS and srsRAN on two different machine and started the gnb after make the configuration change. But sctp is not able to establish. srsRAN ip is reachable from open5Gs vm and Open5gs ip reachable from srsRAN vm.
Open5gs VM ip - 172.16.0.87
AMF config:- ngap ip 172.16.0.87
![image](https://github.com/srsran/srsRAN_Project/assets/144353577/13fe794e-1a8a-4a2c-8a4a-4af2048fd491)

and UPF config:- gtpu ip 172.16.0.87
![image](https://github.com/srsran/srsRAN_Project/assets/144353577/b5fc6e14-610c-4f5f-92f4-55579b1ba891)

gnb VM ip :- 172.16.0.94
gnb_rf_b200_tdd_n78_20Mhz.yml amf: 172.16.0.87
[
![srsRAN](https://github.com/srsran/srsRAN_Project/assets/144353577/b4ea98d2-6e08-4b78-a653-c6fc7ac650dc)
](url)
/
please advice. Thanks

**Comment by aibtw** [comment]
"srsRAN ip is reachable from open5Gs vm and Open5gs ip reachable from srsRAN vm." 

just to clarify, did you try to ping between the two machines? does the ping work from the first machine to the other?

**Comment by pgawlowicz** [comment]
did you set `amf.bind_addr` in the gnb config to an IP that is reachable by Open5GS?

**Reply by rajindersin** [reply, marked as answer]
Thanks for support. I configured the amf.bind_addr in gnb config file as VM IP of gnb and sctp get established
